Sources/Email/EmailAddress.swift

```diff
@@ -0,0 +1,48 @@
+import Vapor
+
+public struct EmailAddress: Content {
+    public let email: String
+    public let name: String?
+    
+    public init(email: String, name: String? = nil) {
+        self.email = email
+        self.name = name
+    }
+}
+
+extension EmailAddress: Emailable {
+    public static var emailKeyPath: KeyPath<EmailAddress, String> {
+        \.email
+    }
+    
+    public static var nameKeyPath: KeyPath<EmailAddress, String?>? {
+        \.name
+    }
+}
+
+extension EmailAddress: ExpressibleByStringLiteral {
+    public init(stringLiteral value: String) {
+        // "From: $Name <$Email>"
+        let regex = try? NSRegularExpression(pattern: #"([a-zA-Z\s]*) <([a-zA-Z@.]*)>$"#, options: .caseInsensitive)
```

Comment:
Since we blindly set the `value` parameter to `email` if the string doesn't match the RegEx, would it be more performant to do some simple string manipulation, split on space, remove greater than and less than symbols from around the email address, and anything else needed to get the values?